core/src/avm2/globals/flash/events/StageVideoEvent.as

```diff
@@ -7,9 +7,27 @@ package flash.events
     
     public class StageVideoEvent extends Event
```

Comment:
Nothing in this class is version-gated; however, the class itself is version-gated (docs say FP 10.2, playerglobals says FP 10.1)